nwg_panel/modules/sway_workspaces.py

```diff
@@ -42,27 +44,17 @@ def build_box(self):
         # prevent from #142
         ws_num = -1
         if self.i3.get_tree().find_focused():
-            ws_num, win_name, win_id, non_empty, win_layout, numbers = self.find_details()
-        
-        if len(self.settings["custom-labels"]) == 1 or len(self.settings["custom-labels"]) == len(self.settings["numbers"]):
-            self.settings["custom-labels"] *= len(self.settings["numbers"])
-        else:
-            self.settings["custom-labels"] = []
-        
-        if len(self.settings["focused-labels"]) == 1 or len(self.settings["focused-labels"]) == len(self.settings["numbers"]):
-            self.settings["focused-labels"] *= len(self.settings["numbers"])
-        else:
-            self.settings["focused-labels"] = []
```

Comment:
these are sanity checks to ensure labels and numbers have the same lengths, maybe better to keep them?